xbmc/Application.cpp

```diff
@@ -252,60 +252,55 @@ using KODI::MESSAGING::HELPERS::DialogResponse;
 //extern IDirectSoundRenderer* m_pAudioDecoder;
 CApplication::CApplication(void)
   : m_pPlayer(new CApplicationPlayer)
+  , m_iScreenSaveLock(0)
```

Comment:
just so we are clear, i mean initialization in the header, ```bool member = false;``` style. order is just as guaranteed (by declaration), but if i add something, i do not have to spend time finding where in the init list it fits. but to each his own, was just a suggestion.